Makefile

```diff
@@ -121,10 +127,11 @@ lint-code: $(golangci_lint_bin)
 
 ##@ Build
 
+# TODO: VERIFY BUILD KEEP WORKING WIHT NEW MAIN!!!!!!!!!!!!!!!
```

Comment:
Did the build keep working 😄 ?

Reply:
hahah forgot to remove the reminders haha